src/main/java/org/resitrack/Main.java

```diff
@@ -60,16 +59,51 @@ public static void managePerson(PersonController personController, Scanner scann
             choice = scanner.nextLine();
 
             switch (choice) {
-                case "1":
-                    personController.addNewPerson();
+                case ADD_PERSON:
+                    System.out.println("Adding new person");
+                    boolean personBeAdded = personController.addNewPerson();
+                    if (personBeAdded)
+                        System.out.println("Add success !");
+                    else
+                        System.out.println("Add false !");
                     break;
-                case "2":
+                case SHOW_ALL_PERSON:
+                    System.out.println("Show all person ");
+                    personController.showAll();
+                    break;
+                case SEARCH_PERSON:
                     personController.searchPeopleByName();
                     break;
+                case DELETE_PERSON:
+                    System.out.println("Delete function: ");
+                    System.out.println("1. Delete person by ID");
+                    System.out.println("2. Delete people by name");
+                    System.out.println("> Enter your choose: ");
+                    String choose = scanner.nextLine();
+                    if (choose.equals("1")) {
+                        System.out.print("Enter IdPerson want to delete: ");
+                        String id = scanner.nextLine();
+                        if(personController.deletePersonById(id)){
+                            System.out.println("Delete success !");
+                        }else {
+                            System.out.println("Delete false !");
+                        }
+                    } else if (choose.equals("2")) {
+                        System.out.print("Enter Name Person want to delete: ");
+                        String name = scanner.nextLine();
+                        if(personController.deletePeopleByName(name)){
+                            System.out.println("Delete success !");
+                        }else {
+                            System.out.println("Delete false !");
+                        }
+                    }else {
+                        System.out.println("Not found ");
+                    }
```

Comment:
I think we should use switch case for code more clean

Reply:
Đã update